src/lib/PPrint.hs

```diff
@@ -425,6 +426,14 @@ instance Pretty (Effect n) where
     RWSEffect rws h -> p rws <+> p h
     ExceptionEffect -> "Except"
     IOEffect        -> "IO"
+    UserEffect name -> "(UserEffect" <+> p name <> ")"
+
+instance Pretty (UEffect n) where
+  pretty eff = case eff of
+    RWSEffect rws h -> p rws <+> p h
+    ExceptionEffect -> "Except"
+    IOEffect        -> "IO"
+    UserEffect name -> "(UserEffect" <+> p name <> ")"
```

Comment:
Given that we print all of those the same way they appear in the surface language, I think we should just print `name` here. And same above